src/sentry/migrations/0180_copy_useroptions_to_notificationsettings.py

```diff
@@ -0,0 +1,124 @@
+# Generated by Django 1.11.29 on 2021-03-15 20:13
+
+from django.db import migrations
+from sentry.models import UserOptionValue
+from sentry.models.integration import ExternalProviders
+from sentry.models.notificationsetting import (
+    NotificationScopeType,
+    NotificationSettingTypes,
+    NotificationSettingOptionValues,
+)
```

Comment:
You should just copy these to the migration instead of importing. This helps keep the migration self contained, so that it doesn't break if we modify these later.